aws-datastore/src/androidTest/java/com/amplifyframework/datastore/storage/SynchronousStorageAdapter.java

```diff
@@ -285,4 +286,14 @@ public void terminate() throws DataStoreException {
             asyncDelegate.observe(emitter::onNext, emitter::onError, emitter::onComplete)
         );
     }
+
+    /**
+     * Invokes the clear method of the underlying adapter and
+     * either completes or throws an exception.
+     */
+    public void clear() {
+        Completable.fromObservable(emitter -> {
```

Comment:
Ah yea, good call. The `Await.result(...)` thing could basically be replaced with a `Single` and a `blockingGet()`. At the time I wrote `Await.result(...)` we hadn't started using `Rx` from the tests across different modules. Here in the DataStore tests, its doubly okay, though, since the actual _source code_ uses Rx.